### PR TITLE
[13.0][ADD] web_widget_many2one_simple

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT27429

preview

### CODE REVIEW

File: web_many2one_simple/README.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    :target: https://translation.odoo-community.org/projects/web-13-0/web-13-0-web_many2one_simple
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds a new widget to many2one fields that allows to link records
+without any extra features (no autocomplete and no search).
```
Comment (pedrobaeza) [inline web_many2one_simple/README.rst]:
```suggestion
without any extra features (no autocomplete and no partial search).
```

File: web_many2one_simple/README.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  * oper: The operator (like, ilike, etc...) [Default is 'ilike']
+
+* no_create: Invalidates the 'can_create' flag
+* no_write: Invalidates the 'can_write' flag
+* noOpen: Disables open the linked record
+
+Usage
+=====
+
+After install this module you need declare the usage of the new widget as follows:
```
Comment (pedrobaeza) [inline web_many2one_simple/README.rst]:
```suggestion
You need to declare the usage of the new widget as follows:
```

File: web_many2one_simple/README.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+``<field name="xxx" widget="many2one_simple">``
```
Comment (pedrobaeza) [inline web_many2one_simple/README.rst]:
```suggestion
``<field name="xxx" widget="many2one_simple" options="...">``
```

**Review by pedrobaeza — COMMENTED** [review]
Any GIF/screenshot?

File: web_many2one_simple/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    "name": "Web One2Many Simple",
+    "summary": """Simple one2many field""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/web",
+    "depends": ["web"],
+    "data": ["views/assets.xml"],
+    "qweb": ["static/src/xml/many2one_simple.xml"],
+    "installable": True,
```
Comment (victoralmau) [inline web_many2one_simple/__manifest__.py]:
```suggestion
    "installable": True,
    "maintainers": ["Tardo"],
```

File: web_many2one_simple/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Tecnativa - Alexandre D. Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Web One2Many Simple",
```
Comment (victoralmau) [inline web_many2one_simple/__manifest__.py]:
```suggestion
    "name": "Web Many2One Simple",
```

File: web_many2one_simple/static/src/js/many2one_simple_field.js [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
+                return Promise.resolve(false);
+            }
+
+            if (this.regex && !this.regex.test(search_value)) {
+                this._showErrorMessage(_t("The given criteria is not valid."));
+                this.reinitialize(false);
+                return Promise.resolve(false);
+            }
+
+            var context = this.record.getContext(this.recordParams);
```
Comment (victoralmau) [inline web_many2one_simple/static/src/js/many2one_simple_field.js]: It's possible to get context from field?

**Comment by Tardo** [comment]
Updated

**Review by victoralmau — APPROVED** [review]

File: web_many2one_simple/README.rst [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+* can_write [Boolean]: Allow edit the linked record
+
+Options:
+~~~~~~~~
+
+* search [Dictionary]
+  * field: The field to use in the search process [Default is 'id']
+  * oper: The operator (like, ilike, etc...) [Default is '=']
+
+* no_create: Invalidates the 'can_create' flag
```
Comment (pedrobaeza) [inline web_many2one_simple/README.rst]: Why this is not using standard names?

File: web_many2one_simple/README.rst [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+* no_write: Invalidates the 'can_write' flag
+* noOpen: Disables open the linked record
```
Comment (pedrobaeza) [inline web_many2one_simple/README.rst]: Why this one is CamelCase instead of snake_case?

File: web_many2one_simple/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Tecnativa - Alexandre D. Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Web Many2One Simple",
+    "summary": """Simple one2many field""",
```
Comment (pedrobaeza) [inline web_many2one_simple/__manifest__.py]:
```suggestion
    "summary": "Simple many2one widget",
```

Comment (pedrobaeza) [reply web_many2one_simple/__manifest__.py]: This one can be the name and without summary

File: web_many2one_simple/README.rst (hunk 7 above)
Comment (Tardo) [reply web_many2one_simple/README.rst]: Follows Odoo implementation

File: web_many2one_simple/README.rst (hunk 8 above)
Comment (Tardo) [reply web_many2one_simple/README.rst]: Follows Odoo implementation

File: web_many2one_simple/__manifest__.py (hunk 9 above)
Comment (Tardo) [reply web_many2one_simple/__manifest__.py]: So.. better rename the whole module to "web_widget_many2one_simple" ?

Comment (pedrobaeza) [reply web_many2one_simple/__manifest__.py]: Well, I didn't mean that, but removing the summary key in the manifest and putting this as name key, but you are totally right that the name of the module is not correct.

File: web_many2one_simple/README.rst (hunk 8 above)
Comment (Tardo) [reply web_many2one_simple/README.rst]: Really can be used in both ways:
- noOpen
- no_open

I updated the readme to use the snake case style

**Comment by Tardo** [comment]
Updated. Module renamed to web_widget_many2one_simple

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**